parsec/backend/postgresql/realm_queries/get.py

```diff
@@ -135,20 +135,21 @@ async def query_get_stats(
 
     if not ret["has_access"]:
         raise RealmAccessError()
-    blocks_size = await conn.fetch(
+    blocks_size_rep = await conn.fetch(
         *_q_get_blocks_size_from_realm(organization_id=organization_id, realm_id=realm_id)
     )
-    vlobs_size = await conn.fetch(
+    vlobs_size_rep = await conn.fetch(
         *_q_get_vlob_size_from_realm(organization_id=organization_id, realm_id=realm_id)
     )
-    RealmStats.blocks_size = 0
-    RealmStats.vlobs_size = 0
-    if "sum" in blocks_size[0] and blocks_size[0]["sum"] is not None:
-        RealmStats.blocks_size = blocks_size[0]["sum"]
-    if "sum" in vlobs_size[0] and vlobs_size[0]["sum"] is not None:
-        RealmStats.vlobs_size = vlobs_size[0]["sum"]
 
-    return RealmStats
+    blocks_size = 0
+    vlobs_size = 0
+    if "sum" in blocks_size_rep[0] and blocks_size_rep[0]["sum"] is not None:
```

Comment:
given we only use the first row, why not do `conn.fetchrow` instead of `blocks_size_rep[0]` ?

Reply:
on top of that, if a row is present, it must contain the `sum` field so not need to check for it presence

Reply:
I did not want to change the logic while messing with the typing.

Should I look into it?

Reply:
11d796eccd0b853c6b597c62a153d149590a8ee0